### DISCUSSION TITLE
[Feature] Add sorting to search results

### DISCUSSION THREAD

**Discussion by schnillerman** [immich-app/immich#10587] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

Add sorting according to attributes available in the search filters to search results, e.g.:
- date
- media type
- file name

### Platform

- [ ] Server
- [X] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
#8377

**Reply by schnillerman** [reply]
Thanks - #8377 doesn't have the keyword sorting in it so it didn't pop up for me when creating this issue.
